app/views/users/edit.html.slim

```diff
@@ -34,6 +34,13 @@
 
   = f.input :name, wrapper_html: { class: 'col-md-6' }
   = f.input :email, required: true, wrapper_html: { class: 'col-md-6' }
+  - unless current_user.confirmed?
+    .help-block
```

Comment:
I wonder if this help text should be more visible. Looking at the screenshot you shared it may kinda get lost in the rest of the info in the page.

Sorry, I know this isn't very actionable. Mentioning it in case someone has a better suggestion.

Reply:
I added a todo to give the help text some design love.

Reply:
Hmm, good point, @pgaspar.

How about using [highlighting](https://getbootstrap.com/docs/3.3/css/#marked-text)? Wdyt?

Reply:
I think it would help 😸